.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,3 +10,5 @@
 - [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
 - [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
 - [ ] The page description includes a link to documentation or a homepage (if applicable).
+
+Version of the command being documented:
```

Comment:
```suggestion
<!-- Please include the version number of the command you have documented (if applicable) here  -->
```
@CleanMachine1 do you mean like this?

Reply:
```suggestion
Version of the command being documented (if known):
```

Reply:
Formatted like this I prefer this version, because it's shorter.